manifests/07_deployment.yaml

```diff
@@ -38,6 +38,8 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
+        - name: CSI_FSGROUP_POLICY
+          value: None
```

Comment:
How exactly will users set this env. var? Right now the deployment is created by cluster-storage-operator and it does not have any option to configure that.

This PR opens a broader question how CSI drivers should be configured. Right now, we don't have any good API for that - `ClusterCSIDriver` does not have any driver-specific fields.

Reply:
I see already discussion above, let's continue there (and please mark this as Resolved)